tests/fuzzer/fuzz/fuzz_targets/bootloader/bootloader_run_single_interaction.rs

```diff
@@ -15,11 +15,7 @@ use rig::forward_system::system::system_types::ForwardRunningSystem;
 use rig::ruint::aliases::{B160, U256};
 use system_hooks::addresses_constants::{
```

Comment:
Note (not related to the PR): I don't think that we testing anything meaningful here, should be revisited later